code/game/objects/items/devices/last_shelter.dm

```diff
@@ -72,7 +72,9 @@ GLOBAL_DATUM(last_shelter, /obj/item/device/last_shelter)
 /obj/item/device/last_shelter/proc/get_cruciform()
 	var/datum/mind/MN = request_player()
 	if(!MN)
-		return FALSE
+		var/obj/item/implant/core_implant/CI = pick_n_take(lost_cruciforms)
+		CI.forceMove(get_turf(src))
+		return CI
```

Comment:
You might want to check if its in an inventory or being held. ForceMoving a object while its in an inventory causes a shadow-item to remain (and gets stuck in the inventory/hand , letting people teleport it to themselves on drop or throw remotely)